extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -155,6 +139,7 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
         trackedSeries.includes(m.referenceSeriesUID)
     );
 
+    // TODO -> Need prompt here.
```

Comment:
Is a prompt for the series description on "export" click part of the ticket?

If so we need to put a prompt here, similar to the `Create Report` button, otherwise we need to create a ticket for this, as you can't current export the SR with a custom SeriesDescription.

Reply:
I've just quicky fixed this so we can push this out.